liquibase-couchbase/src/test/java/liquibase/ext/couchbase/validator/JsonChangelogValidatorTest.java

```diff
@@ -0,0 +1,166 @@
+package liquibase.ext.couchbase.validator;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.ErrorMessageType;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.ValidationMessage;
+import com.networknt.schema.ValidatorTypeCode;
+import liquibase.Scope;
+import liquibase.ext.couchbase.exception.InvalidJSONException;
+import liquibase.ext.couchbase.exception.ResourceNotFoundException;
+import liquibase.resource.Resource;
+import liquibase.resource.ResourceAccessor;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+import java.io.InputStream;
+import java.util.HashSet;
+import java.util.Set;
+
+import static java.util.Collections.emptySet;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@MockitoSettings(strictness = Strictness.LENIENT)
+class JsonChangelogValidatorTest {
+
+    @Mock
+    private ObjectMapper objectMapper;
+    @Mock
+    private JsonSchemaFactory schemaFactory;
+    @InjectMocks
+    private JsonChangelogValidator validator;
+
+    private Scope scope = mock(Scope.class);
+    private ResourceAccessor resourceAccessor = mock(ResourceAccessor.class);
+    private Resource changelogResource = mock(Resource.class);
+    private Resource schemaResource = mock(Resource.class);
```

Comment:
Because these objects are not injected into class level fields. They should be created and used inside methods. But for almost all tests these mocks are the same. So I decided to prepare them on class level for all tests. 

JsonChangeLogValidator has only 2 objects on class level, I used @mock for them and injected them.